lib/themes/dosomething/paraneue_dosomething/includes/auth/login.inc

```diff
@@ -128,12 +128,11 @@ function paraneue_dosomething_pass_submit($form, &$form_state) {
  * @see https://www.owasp.org/index.php/Authentication_Cheat_Sheet#Authentication_and_Error_Messages
  */
 function paraneue_dosomething_pass_set_message() {
-  $default_msg = t(
-    'If your email address exists in our database, you will receive ' .
-    'a password recovery link at your email address in a few minutes.'
-  );
+  $default_msg = 'If your email address exists in our database, you will receive a password recovery link at your email address in a few minutes.';
   $msg = variable_get('dosomething_user_reset_password_copy', $default_msg);
-  drupal_set_message($msg);
+
+  // @TODO: #Global; translation substitution doesn't seem to work correctly with this. Not sure why...?
```

Comment:
Not sure. This is the only one I've had to test so far with the config stuff, but something I'm noting and will need to figure out.